mobile.go

```diff
@@ -104,7 +105,10 @@ func doUpdateMobile(url string, out *os.File, updater Updater) error {
 	}
 
 	req.Header.Add("Accept-Encoding", "gzip")
-	pro.PrepareForFronting(req)
+
+	frontedURL := *req.URL
+	frontedURL.Host = "d2yl1zps97e5mx.cloudfront.net"
+	proxied.PrepareForFronting(req, frontedURL.String())
```

Comment:
Ok, I also verified everything is working with in-app updates. I built a "new" version of the app with an old version number and version code [here](https://github.com/getlantern/lantern-build/blob/pro-backend/lantern-mobile/app/build.gradle#L38), lower than whatever it was for 3.6.3. I was notified an update was available when I opened the app and was able to install it without issue. If you ever need to test mobile in-app updates, make sure to use a *lower version code* of the released version or you might encounter a totally useless `App Not Installed` error (a more informative error appears when you try to install the update manually: `Failure [INSTALL_FAILED_VERSION_DOWNGRADE]`)